.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 matrix:
   include:
-    - node_js: 10
+    - node_js: 12
```

Comment:
Were we still keeping the duplicate environment? I know Chris had talked about removing it since its just creating build cycles with no benefit

Reply:
Agreed: I think we can copy https://github.com/EclipseFdn/iot.eclipse.org/blob/master/.travis.yml for this site.

Reply:
Updated, should I do this to all the other sites that have two builds?